hedera-node/hedera-app/src/test/java/com/hedera/node/app/config/ConfigProviderImplTest.java

```diff
@@ -32,12 +32,6 @@
 @ExtendWith(SystemStubsExtension.class)
 class ConfigProviderImplTest {
 
-    @Test
-    void testNullConfig() {
```

Comment:
Since the class has still `Boolean` as param I would say that it is a good idea to have a test that defines what happens if `null` is passed.

Reply:
Yes, I can remove in a subsequent PR where we remove @Inject and use a little-b boolean.